app/src/test/java/mobile/programming/musicApp/InstrumentDataIntegrationTest.kt

```diff
@@ -32,10 +32,10 @@ class InstrumentDataIntegrationTest {
     }
 
     private fun checkIfInstrumentDataIsEmpty() {
-        var instrumentlist = ArrayList<Instrument>()
+        var instrumentList = ArrayList<Instrument>()
```

Comment:
**Technical Suggestion #3:** According to Kotlin coding conventions, variables and parameters should always be written in lower camelcase. This was probably a simple mistake, but this convention should always be followed for readability purposes. This coding convention can be found under the Variables and Parameters section at: https://github.com/raywenderlich/kotlin-style-guide#variables--parameters